comp/process/processcheck/processcheckimpl/check.go

```diff
@@ -36,6 +37,7 @@ type dependencies struct {
 	Config    config.Component
 	Sysconfig sysprobeconfig.Component
 	WMmeta    workloadmeta.Component
```

Comment:
Curious if it would be better to have Wmeta and Tagger be dependencies of gpuDetector component when it is made into a component (that way I won't have to declare the Tagger in ProcessCheck).

Reply:
That's not a bad idea.

Reply:
Will address in #34137